relayer/tests/runTest.sh

```diff
@@ -26,7 +26,7 @@ sleep 15
 
 
 echo "starting relayer server"
-kill $(lsof -ti :3331) > /dev/null  || true
+kill $(lsof -ti :3332) > /dev/null  || true
```

Comment:
I thought we would't switch this back, since we used 3332 to test a different port.

Reply:
why not just keep 3332?

Reply:
I don't mind, let's just double check that the rest of the code base is consistent.